### PR TITLE
Add cypress+firefox support by upgrading to >= 4.0.0 version

### PR DESCRIPTION
**Pull request by acourdavault**
## Description

As mentioned this add firefox and also incidentally Edge support with cypress.
This is still in Beta but I think bringing this in with the few API changes before we code test that may not be compatible is important with these API changes

### CODE REVIEW

**Comment by acourdavault** [comment]
Todo:
[ ] Check that this works with docker

File: packages/manager/cypress/fixtures/example.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (Jskobos) [inline packages/manager/cypress/fixtures/example.json]: We could probably delete this or ignore it

Comment (acourdavault) [reply packages/manager/cypress/fixtures/example.json]: I ignored it as could be generated by cypress

**Comment by acourdavault** [comment]
@Jskobos I added a sample file for the development.json file
We cannot put comment in json sadly, but this file is for the settings of cypress.
Probably will work on the doc of this as a follow up of what i m doing

**Pull request merged**